HELPING_GUIDE.md

```diff
@@ -0,0 +1,161 @@
+# 📬 Helping Guide
+
+> Anything wrong or missing? Learn how to help Calendarium!
+
+## 🖇️ Forking the repository
+
+**1.** Go to [Calendarium](https://github.com/cesium/calendarium)
+
+**2.** In the top right corner, click on **Fork**
+
+## 🔗 Cloning the forked repository to your machine
+
+**1.** Go to _your fork_ and click **Code**
+
+> **Note**  
+> The link to your forked repository should be `https://github.com/<your-username>/calendarium`
+
+**2.** Copy the given HTTPS link
+
+**3.** On your terminal, run the following command:
+
+```
+git clone <https-link>
+```
+
+> **Note**  
+> If you'd rather clone with SSH, copy the SSH key and run `git clone <ssh-key>`
+
+**Now you have a copy of the repository that you can work with.**
+
+> **Note**  
+> The following instructions are targeted for a Linux environment, you can however use the web-based editing capabilities of GitHub.
+
+## ⛓️ Creating a branch
```

Comment:
Creating a separate branch may not be necessary if you have already forked the repository, it is important to note that in our repository, we adhere to a strict security protocol.

Reply:
i don't see how that is a security problem?

Reply:
I didn't mean't that, i said that in our case we protect main branch due to security. 
If they fork a repository there's no need to create a separate branch on their repository and they can commit directly to main, basically there's no need to create a branch.

Reply:
i see, still i think it makes more sense to create a branch since it's a common practice when contributing to open source projects. i understand it would simplify the process for the person creating the PR but it is also a way for them to learn how to work with git.